src/main/java/gov/usgs/ngwmn/logic/WaterLevelStatistics.java

```diff
@@ -134,15 +96,24 @@ protected void removeMostRecentProvisional(List<WLSample> samples, List<WLSample
 	}
 	
 	// TODO business rule for any one value error? continue without value or error entire statistics calculation?
+	@Override
+	public JsonData calculate(Specifier spec, Reader xmlData) throws Exception {
+		List<WLSample> samples = WLSample.extractSamples(xmlData, spec.getAgencyCd(), spec.getSiteNo(), spec.getElevation());
+		return calculate(spec, samples);
+	}
 
 	@Override
 	public JsonData calculate(Specifier spec, List<WLSample> samples) {
 		LOGGER.trace("Executing WaterLevel Stats calculations.");
 		
 		List<WLSample> samplesByDate = conditioning(spec, samples);
+
 		if (builder.hasErrors()) {
 			return builder.build();
 		}
+		MediationType mediation = findMostPrevalentMediation(spec, samplesByDate);
+		builder.mediation(mediation);
+		
```

Comment:
in later commits or next story, this will have to be controlled a new way. SOGW decided to override this for stats. They want to consistently use BelowLand. It will probably become a parameter.